src/cascadia/TerminalSettingsModel/MTSMSettings.h

```diff
@@ -93,14 +93,14 @@ Author(s):
     X(hstring, TabTitle, "tabTitle")                                                                                                                           \
     X(Model::BellStyle, BellStyle, "bellStyle", BellStyle::Audible)                                                                                            \
     X(IEnvironmentVariableMap, EnvironmentVariables, "environment", nullptr)                                                                                   \
-    X(bool, RightClickContextMenu, "experimental.rightClickContextMenu", false)                                                                                \
+    X(bool, RightClickContextMenu, "rightClickContextMenu", false)                                                                                             \
     X(Windows::Foundation::Collections::IVector<winrt::hstring>, BellSound, "bellSound", nullptr)                                                              \
     X(bool, Elevate, "elevate", false)                                                                                                                         \
     X(bool, AutoMarkPrompts, "autoMarkPrompts", true)                                                                                                          \
     X(bool, ShowMarks, "showMarksOnScrollbar", false)                                                                                                          \
     X(bool, RepositionCursorWithMouse, "experimental.repositionCursorWithMouse", false)                                                                        \
     X(bool, ReloadEnvironmentVariables, "compatibility.reloadEnvironmentVariables", true)                                                                      \
-    X(bool, RainbowSuggestions, "experimental.rainbowSuggestions", false)
+    X(bool, RainbowSuggestions, "rainbowSuggestions", false)

```

Comment:
Personally, I'd prefer if this remained experimental, because I feel like the feature is a little "goofy". It's a neat trick, but I'm not sure if it's serious enough that we should guarantee perpetual support.

Reply:
I do think I agree with this. We need a namespace for "fun goofy features we might remove if they become too expensive"

Reply:
Fair, `experimental` gives us an escape hatch. Reverting